src/inc/conattrs.hpp

```diff
@@ -42,7 +42,8 @@ enum class CursorType : unsigned int
     VerticalBar = 0x1, // A single vertical line, '|'
     Underscore = 0x2, // a single horizontal underscore, smaller that the min height legacy cursor.
     EmptyBox = 0x3, // Just the outline of a full box
-    FullBox = 0x4 // a full box, similar to legacy with height=100%
+    FullBox = 0x4, // a full box, similar to legacy with height=100%
+    DoubleUnderscore = 0x5 // a double horizontal underscore

```

Comment:
I can't remember if these values had a specific VT meaning. @zadjii-msft, @j4james, do you know? Can we just add 0x5 here or did it mean something?

Reply:
Pretty sure these ones are made up values - I'm pretty sure EmptyBox is a cursor shape we made up that's not available in VT

Reply:
All the supported VT styles are defined the `CursorStyle` enum in [DispatchTypes.h](https://github.com/microsoft/terminal/blob/ae550e0969595f062b15c2ff5cc33d4afe8ebc3f/src/terminal/adapter/DispatchTypes.hpp#L317-L326) (not to be confused with the `CursorStyle` enum in TerminalControl). EmptyBox is not currently one of the supported styles, but there was talk of possibly extending the `DECSCUSR` sequence to support others (nosh apparently interprets the values 7 and 8 as an empty box, and 9 and 10 as a star shape).